packages/common-integration-tests/src/storage/storage.ts

```diff
@@ -8,7 +8,7 @@ import {
   StorageGetResponse,
   StoragePutResponse,
 } from '@gomomento/sdk-core';
-import {testStoreName} from './common-int-test-utils';
+import {testStoreName} from '..//common-int-test-utils';
```

Comment:
?

Reply:
I don't know how this still built! Fixed in https://github.com/momentohq/client-sdk-javascript/pull/1410/commits/fb8eec51c612dba7d2b56a60347d5192645b02a7.